react/react-addons-transition-group.d.ts

```diff
@@ -7,7 +7,7 @@
 
 declare namespace __React {
 
-    interface TransitionGroupProps {
+    interface TransitionGroupProps extends HTMLAttributes {
         component?: ReactType;
```

Comment:
This is only true if `component` is `string`, so it would be great if we could model it that way.